tests/test_cuda_bd_exchange_mover.py

```diff
@@ -54,7 +54,7 @@ def test_bd_exchange_validation(precision):
         klass(N, group_idxs, params, DEFAULT_TEMP, beta, cutoff, seed, proposals_per_move, 1)
 
     # Proposals must be non-zero
-    with pytest.raises(RuntimeError, match="proposals per move must be greater than 0"):
+    with pytest.raises(RuntimeError, match="steps per move must be greater than 0"):
```

Comment:
nit: was this change intentional? (looks like the `num_proposals_per_move` that is set to zero here?)

Reply:
Good catch, fixed in [4de6238](https://github.com/proteneer/timemachine/pull/1241/commits/4de62383145c04e5c71e2cca1ce890b4de899881)